### DISCUSSION TITLE
How to retrieve all possible results?

### DISCUSSION THREAD

**Discussion by j1dywfrxvq** [searxng/searx-instances#153] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/110590119/182781829-7093d9e8-ce01-488c-9c85-dc5a76c14c44.png)

**Comment by return42** [comment, marked as answer]
For me https://searx.tuxcloud.net/search?q=123 do work.

FYI: searx.tuxcloud.net is a searx instance (we are SearXNG) that hasn't maintained since a long time.

I recommend to use different instance from https://searx.space/ .. look out for a instance "Version" and select one with with a more recent date.

**Reply by j1dywfrxvq** [reply]
What if nothing works?
https://serx.ml/search?q=123
https://searx.tiekoetter.com/search?q=123
etc...
I want to take an advantage of all results.

**Reply by return42** [reply]
> What if nothing works?

Sorry, I cant reproduce your issue, all the links you gave do work for me. @tiekoetter do you have a clue?

> I want to take an advantage of all results.

What doe you mean by this .. are the result lists you receive are empty?
